### ISSUE TITLE
Checking Kubernetes 1.25 clusters fails due to PodSecurityPolicy

### ISSUE THREAD

**Issue by djablonski-moia** [derailed/popeye#236]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_bug.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**

When checking a cluster that is running Kubernetes 1.25, `popeye` fails with the following error message:

```
Boom! No resource meta found for policy/v1beta1/podsecuritypolicies
```

This is obviously due to the fact that PSPs have been removed from K8s 1.25.

**To Reproduce**

Just run `popeye` without any params against a 1.25 cluster

**Expected behavior**

Results being displayed

**Screenshots**

none

**Versions (please complete the following information):**
 - OS: OSX 12.6.1
 - Popeye 0.10.1
 - K8s 1.25.4

**Additional context**

**Comment by jorgeuos** [comment]
For now, I just commented out the `policy/v1beta1/podsecuritypolicies`. Which is probably not a longterm solution, but at least I can run popeye now.

Diff:
```
diff --git a/pkg/popeye.go b/pkg/popeye.go
index 03f32e4..dc5ad9f 100644
--- a/pkg/popeye.go
+++ b/pkg/popeye.go
@@ -126,7 +126,7 @@ func (p *Popeye) scannedGVRs(rev *client.Revision) []string {
                "apps/v1/replicasets",
                "apps/v1/daemonsets",
                "apps/v1/statefulsets",
-               "policy/v1beta1/podsecuritypolicies",
+               // "policy/v1beta1/podsecuritypolicies",
                "networking.k8s.io/v1/networkpolicies",
                "rbac.authorization.k8s.io/v1/clusterroles",
                "rbac.authorization.k8s.io/v1/clusterrolebindings",
@@ -221,7 +221,7 @@ func (p *Popeye) sanitizers(rev *client.Revision) map[string]scrubFn {
                "apps/v1/statefulsets":                      scrub.NewStatefulSet,
                "networking.k8s.io/v1/networkpolicies":      scrub.NewNetworkPolicy,
                "networking.k8s.io/v1/ingresses":            scrub.NewIngress,
-               "policy/v1beta1/podsecuritypolicies":        scrub.NewPodSecurityPolicy,
+               // "policy/v1beta1/podsecuritypolicies":        scrub.NewPodSecurityPolicy,
                "rbac.authorization.k8s.io/v1/clusterroles": scrub.NewClusterRole,
                "rbac.authorization.k8s.io/v1/clusterrolebindings": scrub.NewClusterRoleBinding,
                "rbac.authorization.k8s.io/v1/roles":               scrub.NewRole,
```

And then:
```
go build && go install
```

Br, Jorge

**Comment by matheusfm** [comment]
I think this issue is resolved in v0.11.1

**Comment by cvallesi-kainos** [comment]
Running clusters on 1.24.9 and wanted to update to 1.25.6.

I'm having the same issue as OP. This was supposed to be fixed but clearly has not. We'd rather not have to use an hack, can someone point out where the logic to be fixed should be in this repo so we may try and propose a PR?
